contrib/babelfishpg_tds/src/include/tds_request.h

```diff
@@ -498,16 +499,20 @@ SetTvpRowData(ParameterToken temp, const StringInfo message, uint64_t *offset)
 }
 
 static inline void
-SetColMetadataForTvp(ParameterToken temp, const StringInfo message, uint64_t *offset)
+SetColMetadataForTvp(ParameterToken temp, const StringInfo message, uint64_t *offset, char *proc_name)
 {
-	uint8_t		len;
-	uint16		colCount;
-	uint16		isTvpNull;
-	char	   *tempString;
-	int			i = 0;
-	char	   *messageData = message->data;
-	StringInfo	tempStringInfo = palloc(sizeof(StringInfoData));
-	uint32_t	collation;
+	uint8_t			len;
+	uint16			colCount;
+	uint16			isTvpNull;
+	char			*tempString;
+	int				i = 0;
+	char	   		*messageData = message->data;
+	StringInfo		tempStringInfo = palloc(sizeof(StringInfoData));
+	uint32_t		collation;
+
+	char			*tvp_type_name = NULL;
+	char 			*tvp_type_schema_name = NULL;
+	char 			*target_arg_name;
```

Comment:
Dont really need any of these new variables in this scope

Reply:
Removed unnecessary arg and moved rest in else block to limit its scope